view/frontend/web/template/payment/google-pay-form.html

```diff
@@ -51,7 +51,7 @@
             <!--/ko-->
         </div>
         <div class="field number googlePay">
-            <div afterRender="renderGooglePay()" data-bind="attr: { id: 'googlePay'}">
+            <div class="checkout-component-dock" afterRender="renderGooglePay()" data-bind="attr: { id: 'googlePay'}">
```

Comment:
what is the class checkout-component-dock doing ?

Reply:
Since now the component is 100% wide we need a container div to control the size of the component. The div was already there so I just added a class for it to be able to style it.